Dart/src/com/jetbrains/lang/dart/projectWizard/DartGeneratorPeer.java

```diff
@@ -172,8 +172,9 @@ private void onTemplatesLoaded(final List<DartProjectTemplate> templates) {
     myCreateSampleProjectCheckBox.setEnabled(true);
     myTemplatesList.setEnabled(true);
 
-    final String selectedTemplateName = PropertiesComponent.getInstance().getValue(DART_PROJECT_TEMPLATE);
-    myCreateSampleProjectCheckBox.setSelected(selectedTemplateName != null);
+    final PropertiesComponent properties = PropertiesComponent.getInstance();
+    final String selectedTemplateName = properties.getValue(DART_PROJECT_TEMPLATE);
+    myCreateSampleProjectCheckBox.setSelected(selectedTemplateName != null || !properties.isValueSet(DART_PROJECT_TEMPLATE));
```

Comment:
It's equivalent to just `setSelected(true)`. This way we enable check box even if it was previously disabled by user.
Do you agree it should be disabled if user has manually disabled it at previous project creation? 
I have fix in hands, just wanted to ask your opinion before pushing.